packages/block-library/src/cover/style.scss

```diff
@@ -196,7 +175,7 @@
 	}
 
 	// Extra specificity for in edit mode where other styles would override it otherwise.
-	img.wp-block-cover__image-background,
```

Comment:
The fixed/repeated background isn't working in the editor in my testing, and I expect it may be because of this.

Reply:
It was a different issue, the classes regarding fixed background were missing in the editor part. Fixed here: https://github.com/WordPress/gutenberg/pull/40554/commits/dbaba8e5e21fb34b66491a66fe295b3288a09ecf